integration/mediation-tests/tests-platform/tests-rabbitmq/src/test/java/org/wso2/carbon/esb/rabbitmq/transport/test/RabbitMQConsumerTestCase.java

```diff
@@ -38,77 +41,67 @@
     public void init() throws Exception {
         super.init();
         initRabbitMQBroker();
+        //The consumer proxy cannot be pre-deployed since the queue declaration(which is done in 'initRabbitMQBroker')
+        // must happen before deployment.
         loadESBConfigurationFromClasspath("/artifacts/ESB/rabbitmq/transport/rabbitmq_consumer_proxy.xml");
```

Comment:
Better to use File.separator in 'loadESBConfigurationFromClasspath' to avoid path traversal.

Reply:
As I remember within loadESBConfigurationFromClasspath replace '/' with File.seperator let's verify it